packages/FlexGrid/Col/Col.jsx

```diff
@@ -5,46 +5,114 @@ import { Subscriber } from 'react-broadcast'
 import { Col as ReactFlexboxGridCol } from 'react-flexbox-grid'
 
 import safeRest from '../../../shared/utils/safeRest'
+import { deprecate } from '../../../shared/utils/warn'
 import styles from './Col.modules.scss'
 
-const removeProps = ({
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
+const removeProps = ({ ...rest }) => safeRest(rest)
```

Comment:
This can be inlined. The `removeProps` function is now redundant. Just use safeRest directly